### PR TITLE
[a11y] Add hints to checkbox component

### PR DESCRIPTION
**Pull request by juanmendes**
Also fixed some a bad previous commit:
* vcdResponsiveInput was added as a class name instead of an attribute
* Module for vcdResponsiveInput was incorrectly setup
* Make sure we only query for input labels and containers that are immediate children of clr-form-control

## Testing Done

- Verified the checkbox input hint's for both the regular checkbox and the toggle buttons.
- Installed this version in VCD's UI to make sure modules were working.

image
image

Signed-off-by: Juan Mendes <jmendes@vmware.com>

### CODE REVIEW

**Comment by irahov** [comment]
For future - better have 2 different commits to address 2 issues: hint & vcdResponsiveInput

File: projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { AfterViewInit, Directive, ElementRef, HostBinding } from '@angular/core
     selector: '.clr-form-control[vcdResponsiveInput]',
 })
 export class ResponsiveInputDirective implements AfterViewInit {
-    @HostBinding('class.clr-row') private clrGridRow = true;
+    @HostBinding('class.clr-row') public clrGridRow = true;
```
Comment (irahov) [inline projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts:18]: why is this needed?

**Review by irahov — APPROVED** [review]

File: projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts (hunk 1 above)
Comment (juanmendes) [reply projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts:18]: Adds `clr-row` to the host of the directive which is required for the responsive CSS behavior.
